did-configuration/demo/public/.well-known/did-configuration

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This is the outer claims object, what I like about this format is that other claim formats can be added later. what I don't like about JWTs is that they are opaque until you decode them.

If this were a linked data signature the claim would be easier to read.